src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -101,7 +101,7 @@ internal static ReadOnlyCollection<CameraDeviceInformation> GetDeviceInformation
         }
 
         internal static CameraDeviceInformation GetDeviceInformation(Native.CameraDeviceStruct device) =>
-            new CameraDeviceInformation(device.Type, device.device, GetString(device.name), GetString(device.id));
+            new CameraDeviceInformation(device.Type, device.device, device.name, device.id, device.extraStreamNum);
 
         private static string GetString(char[] word)
```

Comment:
`GetString()` is deprecated now. It seems possible to delete it.

Reply:
OK. I'll remove `GetString()` method, which is not used anymore.